src/app/core/draft/draft.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@angular/core';
+import { HttpService } from '../http/http.service';
+import { AppConfig } from '../../app.config';
+import { HttpErrorService } from '../http/http-error.service';
+import { CaseEventData } from '../../shared/domain/case-event-data';
+import { Observable } from '../../../../node_modules/rxjs';
+import { Draft } from '../../shared/domain/draft';
+
+@Injectable()
+export class DraftService {
+  constructor(
+    private http: HttpService,
+    private appConfig: AppConfig,
+    private errorService: HttpErrorService
+  ) {}
+
+  // getDrafts(jid: string, ctid: string): Observable<Draft> {
```

Comment:
Should we keep it on a branch or stashed rather than remember to uncomment it later?